src/components/Heading/Heading.module.scss

```diff
@@ -1,3 +1,4 @@
 .heading {
   line-height: var(--size-line-height-heading);
+  font-family: Nexa, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
should this come from a token instead?

Reply:
tokens for fonts are not available at the moment.